crates/bevy_asset/Cargo.toml

```diff
@@ -30,7 +30,7 @@ anyhow = "1.0.4"
 thiserror = "1.0"
 downcast-rs = "1.2.0"
 notify = { version = "=5.0.0-pre.11", optional = true }
-parking_lot = "0.11.0"
+parking_lot = "0.12.1"
```

Comment:
IMO it'd be better to not pin to a patch version and just accept any semver compatible version via "0.12".

Reply:
putting 0.12 would allow 0.12.0 which I didn't test, 0.12.1 will still be ok with 0.12.x as long as it's after 0.12.1